.github/workflows/gh-pages.yml

```diff
@@ -42,6 +42,9 @@ jobs:
       - name: Run jest
         run: rm -rf docs/coverage && yarn test --coverage
 
+      - name: Make coverage badge
+        run: npx make-coverage-badge
```

Comment:
would it be safer to use a specific tagged version rather than relying on the latest?

Reply:
Probably safer, but would only break the badge—will do if it breaks